stack_data/formatting.py

```diff
@@ -138,6 +138,8 @@ def format_frame(self, frame: Union[FrameInfo, FrameType, TracebackType]) -> Ite
         for line in frame.lines:
             if isinstance(line, Line):
                 yield self.format_line(line)
+            elif isinstance(line, BlankLines) and self.show_linenos:
```

Comment:
This implies that `not options.skip_blank and not self.show_linenos` may or may not fail on `assert_(line is LINE_GAP)`. If that combination shouldn't be allowed, it should be checked in `__init__`.

Reply:
Ah, yes, good point.